src/main/java/org/hydev/mcpm/client/database/fetcher/LocalDatabaseFetcher.java

```diff
@@ -153,7 +155,7 @@ private String readDatabaseFromContent(HttpEntity entity, DatabaseFetcherListene
         listener.finish();
 
         // Decompress ZSTD
-        if (this.enableCompression)
```

Comment:
Oh god... I think this is an IntelliJ warning (checking a value that's always true). Did you silence it?

Reply:
hmm it's actually not a warning. maybe the compiler optimizes it 🤔

Reply:
Looks like it! My bad. Whether or not the compiler optimizes it, I've been warned for this before (I promise it was `static final bool`), so I'm glad they fixed whatever madness was causing it.